include/nuttx/clock.h

```diff
@@ -394,6 +394,49 @@ void clock_resynchronize(FAR struct timespec *rtc_diff);
 clock_t clock_systime_ticks(void);
 #endif
 
+/****************************************************************************
+ * Name: clock_time2ticks
+ *
+ * Description:
+ *   Return the given struct timespec as systime ticks
+ *
+ *   NOTE:  This is an internal OS interface and should not be called from
+ *   application code.
+ *
+ * Input Parameters:
+ *   reltime - Time presented as struct timespec
```

Comment:
```suggestion
 * Input Parameters:
 *   reltime - Time presented as struct timespec
 *
 * Output Parameters:
 *   ticks   - Pointer to receive the time value presented as systime ticks
```
It might be interesting to also add the documentation for the output parameters.